<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Текстовый Редактор</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="openRtfAction"/>
    <addaction name="createRtfAction"/>
    <addaction name="saveRtfAction"/>
   </widget>
   <widget class="QMenu" name="editMenu">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="findAction"/>
    <addaction name="replaceAction"/>
    <addaction name="clearAction"/>
    <addaction name="undoAction"/>
    <addaction name="copyAction"/>
    <addaction name="pasteAction"/>
    <addaction name="insertTableAction"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="editMenu"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>Основные действия</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="openRtfAction"/>
   <addaction name="createRtfAction"/>
   <addaction name="saveRtfAction"/>
   <addaction name="separator"/>
   <addaction name="findAction"/>
   <addaction name="replaceAction"/>
   <addaction name="clearAction"/>
   <addaction name="separator"/>
   <addaction name="undoAction"/>
   <addaction name="copyAction"/>
   <addaction name="pasteAction"/>
   <addaction name="insertTableAction"/>
  </widget>
  <widget class="QToolBar" name="controlPanel">
   <property name="windowTitle">
    <string>Панель управления</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="openRtfAction">
   <property name="text">
    <string>Открыть .rtf</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="createRtfAction">
   <property name="text">
    <string>Создать .rtf</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="saveRtfAction">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Сохранить .rtf</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="findAction">
   <property name="text">
    <string>Найти</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="replaceAction">
   <property name="text">
    <string>Заменить</string>
   </property>
  </action>
  <action name="clearAction">
   <property name="text">
    <string>Очистить</string>
   </property>
  </action>
  <action name="undoAction">
   <property name="text">
    <string>Вернуть</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="copyAction">
   <property name="text">
    <string>Копировать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="pasteAction">
   <property name="text">
    <string>Вставить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="insertTableAction">
   <property name="text">
    <string>Вставить таблицу</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
